### PR TITLE
hold my beer while we hope this works.

### PR DESCRIPTION
**Pull request by j2m6lz5n**

### CODE REVIEW

File: scripts/busca.js [hunk 1]
```diff
@@ -76,17 +76,24 @@ module.exports = function(robot) {
         res.reply(':thinking_face: I can\'t find any upcoming events for ' + name + '.');
         return;
       }
-
+      edates = [];
```
Comment (thejandroman) [inline scripts/busca.js]: This declares a global variable. We should limit to local function scope. Consider `var edates = [];` to limit to the local scope.

File: scripts/busca.js [hunk 2]
```diff
@@ -76,17 +76,24 @@ module.exports = function(robot) {
...
-
       eventKeys.forEach(function(key) {
         var event = data[key],
             summary = event.summary,
             startDate = event.start.toDateString(),
             endDate = event.end.toDateString();
-        response += '\n- ' + summary + ' from ' + startDate + ' - ' + endDate;
+            edates.push(event);
+
+        //response += '\n- ' + summary + ' from ' + startDate + ' until ' + endDate;
```
Comment (thejandroman) [inline scripts/busca.js]: We can go ahead and remove this comment. It doesn't clarify any code and we can grab the history directly from git.

File: scripts/busca.js [hunk 3]
```diff
@@ -76,17 +76,24 @@ module.exports = function(robot) {
...
         return;
       }
-
+      edates = [];
       response = ':spiral_calendar_pad: Looks like ' + name
         + ' has the following ' + numEvents + ' events:';
-
       eventKeys.forEach(function(key) {
         var event = data[key],
             summary = event.summary,
```
Comment (thejandroman) [inline scripts/busca.js]: We can remove some of these variables since they're no longer used.

File: scripts/busca.js (hunk 1 above)
Comment (j2m6lz5n) [reply scripts/busca.js]: cool...done

File: scripts/busca.js (hunk 2 above)
Comment (j2m6lz5n) [reply scripts/busca.js]: done...

File: scripts/busca.js (hunk 3 above)
Comment (j2m6lz5n) [reply scripts/busca.js]: testing now

**Review by thejandroman — CHANGES_REQUESTED** [review]
This looks fine. I've added some comments that we should consider changing before merging.

File: scripts/busca.js [hunk 4]
```diff
@@ -76,17 +76,19 @@ module.exports = function(robot) {
...
       response = ':spiral_calendar_pad: Looks like ' + name
         + ' has the following ' + numEvents + ' events:';
-
       eventKeys.forEach(function(key) {
-        var event = data[key],
-            summary = event.summary,
-            startDate = event.start.toDateString(),
-            endDate = event.end.toDateString();
-        response += '\n- ' + summary + ' from ' + startDate + ' - ' + endDate;
+        var event = data[key]
```
Comment (thejandroman) [inline scripts/busca.js]: Don't forget the trailing semi-colon (`;`) here.

**Review by thejandroman — APPROVED** [review]

**Pull request merged**